README.md

```diff
@@ -1023,7 +1023,7 @@ For instance:
 let dateRow : DateRow? = form.rowBy(tag: "dateRowTag")
 let labelRow: LabelRow? = form.rowBy(tag: "labelRowTag")
 
-let dateRow2: Row<Date>? = form.rowBy(tag: "dateRowTag")
+let dateRow2: Row<DateCell>? = form.rowBy(tag: "dateRowTag") as? Row<DateCell>
```

Comment:
@m-revetria @mcousillas6  According Swift type inference we should not need need `as? Row<DateCell>` here.